### PR TITLE
Add node and php to benchmark dashboard

### PR DESCRIPTION
**Pull request by BSBandme**

### CODE REVIEW

File: benchmarks/util/result_uploader.py [hunk 1]
```diff
@@ -59,13 +59,14 @@ def upload_result(result_list, metadata):
...
-
-    bq = big_query_utils.create_big_query()
-    row = big_query_utils.make_row(str(uuid.uuid4()), new_result)
-    if not big_query_utils.insert_rows(bq, _PROJECT_ID, _DATASET,
-                                       _TABLE + "$" + _NOW,
-                                       [row]):
-      print('Error when uploading result', new_result)
+    print(labels_string)
+# 
+#     bq = big_query_utils.create_big_query()
```
Comment (TeBoring) [inline benchmarks/util/result_uploader.py]: Is this intentionally commented out?

Comment (BSBandme) [reply benchmarks/util/result_uploader.py]: Sorry that I forgot to uncomment this after testing.
Done.

**Comment by BSBandme** [comment]
[Dashboard](http://performance-dot-grpc-testing.appspot.com/explore?dashboard=5715350830186496&widget=596013819&container=581906293)

File: benchmarks/util/result_parser.py [hunk 2]
```diff
@@ -252,5 +284,11 @@ def get_result_from_file(cpp_file="",
     __parse_go_result(go_file)
   if synthetic_file != "":
     __parse_synthetic_result(synthetic_file)
+  if node_file != "":
+    __parse_js_php_result(node_file, "node")
```
Comment (TeBoring) [inline benchmarks/util/result_parser.py]: Why php and js share the same method?

Comment (BSBandme) [reply benchmarks/util/result_parser.py]: Their results have the same json format (which are constructed by myself). For code reuse I just use the same method to parse the result.

Comment (TeBoring) [reply benchmarks/util/result_parser.py]: Could you give it a better name? e.g., __parse_XX_format_result and let js and php wrap it?
Why other parse methods work with different json format?

Comment (BSBandme) [reply benchmarks/util/result_parser.py]: Done. The other results are most like using third_party benchmark tools for benchmark, e.g. c++ is using google/benchmark, the json result format are defined by the tool. But js and php and just using language internal time library to get the consuming time.

File: benchmarks/util/result_parser.py [hunk 3]
```diff
@@ -236,11 +234,51 @@ def __parse_go_result(filename):
...
+#   {
+#     "filename": string,
+#     "benchmarks": {
+#       behavior: results,
+#       ...
+#     },
+#   },
+#   ...
+# ]
+def __parse_js_php_result(filename, language):
```
Comment (TeBoring) [inline benchmarks/util/result_parser.py]: Could you change a name? js_php tells nothing about the format.

Comment (TeBoring) [reply benchmarks/util/result_parser.py]: __parse_custom_result?

**Pull request merged**